app/jobs/spending_breakdown_job.rb

```diff
@@ -3,10 +3,15 @@ class SpendingBreakdownJob < ApplicationJob
 
   def perform(requester_id:, fund_id:)
     requester = User.find(requester_id)
+    source_fund = Fund.new(fund_id)
 
-    export = Export::SpendingBreakdown.new(source_fund: Fund.new(fund_id))
+    export = Export::SpendingBreakdown.new(source_fund: source_fund)
     upload = upload_csv_to_s3(file: save_tempfile(export), filename: export.filename)
 
+    fund = source_fund.activity
```

Comment:
Probably not a big deal, but do you think it's worth referring to this as a `fund_activity` rather than simply `fund` as there's a bit of a discrepancy with the `Fund` class above?